docs/api/portable-stories.md

```diff
@@ -0,0 +1,184 @@
+---
+title: 'Portable stories'
+---
+
+TK - Intro
+
+## composeStories
+
+`composeStories` will process all stories from the component's stories you specify, compose args/decorators in all of them and return an object containing the composed stories.
+
+If you use the composed story (e.g. Primary button), the component will render with the args that are passed in the story. However, you are free to pass any props on top of the component, and those props will override the default values passed in the story's args.
+
+```tsx
+// Button.test.ts|tsx
+import { render, screen } from '@testing-library/react';
+import { composeStories } from '@storybook/react';
+import { test, expect } from 'vitest';
+// import all stories from the stories file
+import * as stories from './Button.stories';
+
+// Every component that is returned maps 1:1 with the stories,
+// but they already contain all decorators from story level, meta level and project level.
+const { Primary, Secondary } = composeStories(stories);
+
+test('renders primary button with default args', () => {
+  render(<Primary />);
+  const buttonElement = screen.getByText('Text coming from args in stories file!');
+  expect(buttonElement).not.toBeNull();
+});
+
+test('renders primary button with overriden props', () => {
+  // You can override props and they will get merged with values from the Story's args
+  render(<Primary>Hello world</Primary>);
+  const buttonElement = screen.getByText(/Hello world/i);
+  expect(buttonElement).not.toBeNull();
+});
+```
+
+### Type
+
+<!-- prettier-ignore-start -->
+```ts
+(
+  csfExports: CSF file exports,
+  projectAnnotations?: ProjectAnnotations
+) => Record<string, ComposedStoryFn>
+```
+<!-- prettier-ignore-end -->
+
+### Parameters
+
+#### `csfExports`
+
+(**Required**)
+
+Type: CSF file exports
+
+Specifies which component's stories you want to compose. Pass the **full set of exports** from the CSF file (not the default export!).
```

Comment:
will people understand "full set of exports"? would it be better to add an example snippet like so?

```ts
import * as stories from './Button.stories';
```